zerver/views/invite.py

```diff
@@ -232,6 +233,46 @@ def resend_user_invite_email(
     return json_success(request)
 
 
+@require_member_or_admin
+@typed_endpoint
+def edit_multiuse_invite(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    *,
+    invite_id: PathOnly[int],
+    stream_ids: Json[list[int]] | None = None,
+    invite_as: Annotated[
+        Json[int] | None,
+        check_int_in_validator(list(PreregistrationUser.INVITE_AS.values())),
+    ] = None,
+    include_realm_default_subscriptions: Json[bool] | None = None,
+) -> HttpResponse:
+    try:
+        invite = MultiuseInvite.objects.get(id=invite_id)
+    except MultiuseInvite.DoesNotExist:
+        raise JsonableError(
+            _("Invite does not exist with id: {invite_id}.").format(invite_id=invite_id)
+        )
```

Comment:
There is no check for permission to create multiuse invites. Did we discuss to remove that check?

Reply:
As far as I can understand, as per [#api design > edit multiuse invites @ 💬](https://chat.zulip.org/#narrow/channel/378-api-design/topic/edit.20multiuse.20invites/near/2158770), we would not allow a user to edit invites if they cannot create. And you have that check in frontend as well.